README.md

```diff
@@ -12,7 +12,7 @@ Matterpoll is a plugin for [Mattermost](https://mattermost.com/). It allows user
 ## Installation
 
 1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+2. Upload this file in the Mattermost **System Console > Plugins > Plugin Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#custom-plugins).
```

Comment:
I hope we can simplify this to 
`In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.`

Reply:
Great 👍